core/frontend/src/quantity-formatting/LocalUnitFormatProvider.ts

```diff
@@ -0,0 +1,79 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Utils

```

Comment:
Update to `Quantity Formatting` module.

Reply:
Changed to QuantityFormatting as no other module name contain a space.

Reply:
Sounds good forgot about that restriction.